API_v1.md

```diff
@@ -689,15 +927,15 @@ Create a new user with a place and a contact.
 
 ### Permissions
 
-`can_create_users`  
+`can_create_users`, `cat_create_places`, `cat_create_contacts`
```

Comment:
Make sure these permissions are defined in medic-webapp/packages/kujua-sms/views/lib/app_settings.js otherwise there's no easy way to set them.